docs/source/getting_started/install.rst

```diff
@@ -1,76 +1,118 @@
-`Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__
+Please clone `DeepReg`_ first and change current directory to the DeepReg root directory:
 
-DeepReg primarily supports and is regularly tested with Ubuntu and Debian Linux distributions.
+.. code:: bash
+
+    git clone git@github.com:DeepRegNet/DeepReg.git
+    cd DeepReg
+
+then, install or update the conda environment following the instructions below.
+Please see the `official conda documentation <https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html>`__
+for more details.
 
 .. tabs::
 
     .. tab:: Linux
 
-        With CPU only
+        Install DeepReg without GPU support.
 
         .. code:: bash
 
             conda env create -f environment_cpu.yml
             conda activate deepreg
 
-        With GPU
+        Install DeepReg with GPU support.
 
         .. code:: bash
 
             conda env create -f environment.yml
             conda activate deepreg
 
+        Update DeepReg without GPU support.
+
+        .. code:: bash
+
+            conda env update -f environment_cpu.yml
+
+
+        Update DeepReg with GPU support.
+
+        .. code:: bash
+
+            conda env update -f environment.yml
 
     .. tab:: Mac OS
 
-        With CPU only
+        Install DeepReg without GPU support.
 
         .. code:: bash
 
             conda env create -f environment_cpu.yml
             conda activate deepreg
 
+        Update DeepReg without GPU support.
+
+        .. code:: bash
 
-        With GPU
+            conda env update -f environment_cpu.yml
+
+        Install/update DeepReg with GPU support.
 
         .. warning::
 
             Not supported or tested.
 
     .. tab:: Windows
 
-        With CPU only
+        Install/update DeepReg without GPU support.
 
         .. warning::
 
-            DeepReg on Windows is not fully supported. However, you can use the `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__
-            with CPU only. Set up WSL and follow the DeepReg setup instructions for Linux.
+            DeepReg on Windows is not fully supported.
+            However, you can use the `Windows Subsystem for Linux <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__.
+            Set up WSL and follow the DeepReg setup instructions for Linux.
 
-        With GPU
+        Install/update DeepReg with GPU support.
 
         .. warning::
 
             Not supported or tested.
 
-Use docker
-----------
+
+After activating the conda environment, please install DeepReg locally:
+
+.. code:: bash
+
+    pip install -e .
+
+Install via docker
+------------------
 
 We also provide the docker file for building the docker image.
+Please clone `DeepReg repository`_ first:
+
+.. code:: bash
+
+    git clone git@github.com:DeepRegNet/DeepReg.git
+
+then install DeepReg following the instructions below.
 
 Install docker
```

Comment:
Typo in line 142:

docker run --name <container_name> --privileged=true -ti deepreg bash

should be 

docker run --name <container_name> --privileged=true -it deepreg bash